core/class-btm-plugin-options.php

```diff
@@ -514,4 +515,51 @@ public function update_delete_old_entities_cron_job_interval_in_days( $interval
 
 		return false;
 	}
+
+
+	/**
+	 * @var string
+	 */
+	private $show_system;
+
+	/**
+	 * @return string
+	 */
+	public function get_show_system(){
+		return $this->show_system;
+	}
+	/**
+	 * @param string $show_system
+	 */
+	public function set_show_system( $show_system ){
+		$this->show_system = $show_system;
+	}
+	/**
+	 * Initializes the show system tasks
+	 */
+	private function init_show_system(){
+		$show_system = get_option( $this->get_db_table_prefix() . 'show_system' , 'off' );
```

Comment:
@eriktad seems like here we are using dynamic prefix, it's a bit strange that we are using table prefix as option name prefix, are we doing the same for other options as well?